dev/NavigationView/NavigationView.xaml

```diff
@@ -285,6 +285,40 @@
                                                         <Setter Property="Padding" Value="{ThemeResource TopNavigationViewOverflowMenuPadding}" />
                                                         <!-- Set negative top margin to make the flyout align exactly with the button -->
                                                         <Setter Property="Margin" Value="0,-4,0,0" />
+                                                        <Setter Property="ScrollViewer.HorizontalScrollMode" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.VerticalScrollMode" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.VerticalScrollBarVisibility" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.ZoomMode" Value="Disabled" />
+                                                        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" />
```

Comment:
><contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" /> [](start = 56, length = 95)

Do we need the contract7NotPresent setter?

Reply:
We don't need it here because there is nothing really to set it to. The actually important contract7NotPresent setter is this one: https://github.com/microsoft/microsoft-ui-xaml/blob/b9d5ae1738e5d720ef7aecefbce9dc7a059a1f28/dev/NavigationView/NavigationView.xaml#L317

The `FlyoutPresenter.CornerRadius` API is only available since 1809 so to get rounded corners on earlier Windows 10 versions, we use the contract7NotPresent setter above.